storage/persistent_storage/persistent-storage-ocs.adoc

```diff
@@ -23,73 +23,73 @@ OpenShift Container Storage on top of Red Hat Hyperconverged Infrastructure (RHH
 2+^| *Planning*
 
 |What's new, known issues, notable bug fixes, and Technology Previews
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html/4.7_release_notes/[Red Hat OpenShift Container Storage 4.7 Release Notes]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.8/html/4.8_release_notes/[Red Hat OpenShift Container Storage 4.8 Release Notes]
 
 |Supported workloads, layouts, hardware and software requirements, sizing and scaling recommendations
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html/planning_your_deployment/index[Planning your Red Hat OpenShift Container Storage 4.7 deployment]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.8/html/planning_your_deployment/index[Planning your Red Hat OpenShift Container Storage 4.8 deployment]
 
 2+^| *Deploying*
 
 //|Preparing to deploy when your environment is not directly connected to the internet
-//|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html/preparing_to_deploy_in_a_disconnected_environment/[Preparing to deploy OpenShift Container Storage 4.7 in a disconnected environment]
+//|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.8/html/preparing_to_deploy_in_a_disconnected_environment/[Preparing to deploy OpenShift Container Storage 4.8 in a disconnected environment]
```

Comment:
This content seems to have moved to https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.8/html/planning_your_deployment/disconnected-environment_rhocs

Reply:
acked